maturity-level-two/README.md

```diff
@@ -11,3 +11,50 @@ You should:
 - Add your OpenAPI specs to source control given this is part of your application
 - Validate changes to your OpenAPI specs to avoid specification violations ([user guide](docs/validating-open-api-specs.md))
 - Unit test Open API validation to automatically detect breaking changes
+
+ ## Content negotiation
+ When no specific compatibility requirements regarding the rest request and response formats are set, it is recommended to use the JSON format (application/json). But in some situations a client might require the data to be formatted in a certain format in order to interpret the data correctly. Herefore content negotiation may be required, in which the client specifies in which format a request may be send and in which response formats the client can accept. By default ASP.NET Core web applications will expect JSON requests and send JSON responses. In addition to this, ASP.NET Core also uses some additional formatters for special cases, such as the TextOutputFormatter and the HttpNoContentOutputFormatter.
```

Comment:
Frankly, I'm missing guidance on when to use the attributes over the default formatters as those feel like the best place to start